<launch> 
   
  <arg name="launch_prefix" default="" /> <!-- set to value="gdbserver localhost:10000" for remote debugging -->
  <arg name="node_namespace" default="cup_find" />
  <arg name="camera_name" default="/camera" />
  <arg name="camera_frame" default="camera_link" />
  <arg name="image_topic" default="rgb/image_raw" />
  <arg name="is_continue" default="true" />

  <!-- Set parameters --> 
  
  <node pkg="find_cup_ros" type="find_cup_ros_continuous_node" name="$(arg node_namespace)" clear_params="true" output="screen" launch-prefix="$(arg launch_prefix)" >
    <!-- Remap topics from those used in code to those on the ROS network -->
    <remap from="image_rect" to="$(arg camera_name)/$(arg image_topic)" />
    <remap from="camera_info" to="$(arg camera_name)/camera_info" />
    <param name="cup_height" type="double" value= "0.0476"/>
    <param name="beta" type="double" value= "0.1"/>
    <param name="camera_frame" type="str" value="$(arg camera_frame)" /> 
    <param name="is_continue" type="bool" value="$(arg is_continue)" /> 
  </node>
 
</launch>


